scripts/behavior/behavior_test.py

```
@@ -0,0 +1,191 @@
+
+    def _recv_exact(self, n):
+        data = b""
+        while len(data) < n:
+            chunk = self.sock.recv(n - len(data))
+            if not chunk:
+                raise ConnectionError("RCON connection closed")
+            data += chunk
+        return data
+
+    def _auth(self, password):
+        rid, ptype, _ = self._send(3, password)
```

Ruff (0.16.1)

[warning] Unpacked variable `rid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)